com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/ShaderUtilities.cs

```diff
@@ -3,12 +3,14 @@
 
 using Microsoft.CSharp;
 using System;
+using System.Drawing;
```

Comment:
Could you remove the System.Drawing and UnityEditor.ShaderData usings?

This happens to me all the time, luckily Visual Studio gives you a hint that usings aren't required by greying them out slightly:
![image](https://user-images.githubusercontent.com/13305729/216201054-a67adbab-b8ad-4f36-a01a-55e402bcad55.png)